README.rst

```diff
@@ -75,10 +75,12 @@ Compatibility Notes
 +------------------+---------------+
```

Comment:
Since we're removing Python 3.8 support, we need to add the version limit (< 12.0.0) to the compatibility table for versions Quince and below, in case it's not already specified.
https://github.com/eduNEXT/eox-core/blob/ac71f2bfba1153fb56e6d5480d53a5412827a12e/README.rst#compatibility-notes